src/Lockedfield.php

```diff
@@ -251,7 +251,7 @@ public function getForbiddenStandardMassiveAction()
 
     public static function canPurge()
     {
-        return Session::haveRight(static::$rightname, UPDATE);
+        return Session::haveRight(static::$rightname, PURGE);
     }
```

Comment:
This method is probably useless now, as the default implementation can be used now.

Reply:
indeed